### ISSUE TITLE
Code on 'master' branch does not compile when Beagle library used.

### ISSUE THREAD

**Issue by kusalananda** [NBISweden/MrBayes#1]
Seems to be related to large commit on the 28th of December, 2015.

**Comment by kusalananda** [comment]
I'm trying to fix this on the "feature/fixup" branch, off the "develop" branch.

**Comment by ronquist** [comment]
See my pull request that will solve this.

**Comment by ronquist** [comment]
This pull request: #10

**Comment by kusalananda** [comment]
Copying my comment on the pull request into here instead:

On OS X with the `clang` compiler, I get:

```
/usr/bin/clang -DHAVE_CONFIG_H -I.    -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -maes -mavx -mfma -mavx2 -mrdrnd -mbmi -mbmi2 -I/Users/kk/local/stow/beagle-lib/include/libhmsbeagle-1   -g -O2 -Wall -MT mb-command.o -MD -MP -MF .deps/mb-command.Tpo -c -o mb-command.o `test -f 'command.c' || echo './'`command.c
In file included from command.c:40:
./mbbeagle.h:24:14: error: conflicting types for 'beagleGetVersion'
extern char *beagleGetVersion (void);
             ^
/Users/kk/local/stow/beagle-lib/include/libhmsbeagle-1/libhmsbeagle/beagle.h:240:30: note:
      previous declaration is here
BEAGLE_DLLEXPORT const char* beagleGetVersion(void);
                             ^
1 error generated.
```


Also with `gcc` on Linux:

```
gcc -DHAVE_CONFIG_H -I.    -mmmx -msse -msse2 -msse3 -mssse3 -msse4.1 -msse4.2 -msse4a -maes -mavx -mfma -mfma4 -mxop -mbmi -mabm -I/home/andkaha/local/stow/beagle-lib/include/libhmsbeagle-1   -g -O2 -Wall -MT mb-command.o -MD -MP -MF .deps/mb-command.Tpo -c -o mb-command.o `test -f 'command.c' || echo './'`command.c
In file included from command.c:40:
mbbeagle.h:24: error: conflicting types for ‘beagleGetVersion’
/home/andkaha/local/stow/beagle-lib/include/libhmsbeagle-1/libhmsbeagle/beagle.h:240: note: previous declaration of ‘beagleGetVersion’ was here
make[2]: *** [mb-command.o] Error 1
```

This comes from  ```extern const char *beagleGetVersion (void);``` in `mbbeagle.h`.

This `extern` declaration is not needed as the all C code includes `bayes.h` which in turn includes the `<libhmsbeagle/beagle.h>` header where this particular function is actually defined.

**Comment by kusalananda** [comment]
I would also like to know what compiler and architecture you're on, and if you set any `CCFLAGS` when running `configure`.  I don't have access to a C11-conformant compiler... The best I can do is C99.

Do we want to have `-std=c99 -pedantic` as a default option? What should happen if `-std=c99` is not supported?

**Comment by kusalananda** [comment]
Merged Fredrik's pull request #10. This closes this issue.
